addons/utm/models/utm_medium.py

```diff
@@ -20,10 +20,11 @@ class UtmMedium(models.Model):
 
     @api.model_create_multi
     def create(self, vals_list):
-        new_names = self.env['utm.mixin']._get_unique_names(self._name, [vals.get('name') for vals in vals_list])
-        for vals, new_name in zip(vals_list, new_names):
-            vals['name'] = new_name
-        return super().create(vals_list)
+        new_names = []
+        for item in vals_list:
+            if item.get('name'):
+                new_names.append({'name': item['name'].lower()})
+        return super().create(new_names)
```

Comment:
Wouldn't this mean we lose all values except for the name?
We should modify `vals_list` or create new values from it.

So maybe
```
for vals in vals_list:
    if name := vals.get('name')
        vals['name'] = name.lower()
```